iron-a11y-keys-base.html

```diff
@@ -0,0 +1,425 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer-element.html">
```

Comment:
I think it would cause 1.x elements to produce a 404 error, but this shouldn't prevent it from functioning?

The bigger issue is that this uses ES6 code which would be a breaking change for 1.x.  I don't think this PR is compatible with 1.x.